Formula/gq-cli.rb

```diff
@@ -0,0 +1,20 @@
+class GqCli < Formula
+  desc "Command-line interface for GQ"
+  homepage "https://goquant.io"
+  url "https://brickelldatabros1aboah.blob.core.windows.net/public/gq-cli-source-0.0.4.tar.gz"
+  sha256 "6e2b86cf57035bba045d7db9f1d28ae83283cec6b0ed4335083f320374b951ba"
+  license "MIT"
```

Comment:
Where does this license information come from?

Reply:
From what I have been told this a private MIT license